.gitignore

```diff
@@ -59,3 +59,11 @@ typings/
 
```

Comment:
Why is the node-modules folder not included in here?

Reply:
node-modules should be in `.git-ignore`, and get installed on `npm install`

Keep in mind, this is the development code of interspect. I assume that somewhere in the production pipeline, the nodemodules are incorporated into the electron app.